packages/base/src/utils/constants.ts

```diff
@@ -430,6 +430,13 @@ export const WBTC_MAP: Record<EverythingButBase, TokenWithMarketId> = {
   },
 };
 
+export const METH_MAP: Record<Network.Mantle, TokenWithMarketId> = {
+  [Network.Mantle]: {
+    address: '0xcDA86A272531e8640cD7F1a92c01839911B90bb0',
+    marketId: 100, // @follow-up Not sure what marketId
```

Comment:
As of now, no assets have been added yet so there is no market ID. Please change this to `-1` for now and change this to `TODO`